pom.xml

```diff
@@ -65,7 +65,7 @@
     <shadeBase>net.snowflake.ingest.internal</shadeBase>
     <slf4j.version>1.7.36</slf4j.version>
     <snappy.version>1.1.10.4</snappy.version>
-    <snowjdbc.version>3.13.30</snowjdbc.version>
+    <snowjdbc.version>3.14.5</snowjdbc.version>
```

Comment:
yeah, we can wait now that it is pushed. but we will have to wait til next release of JDBC. 
are we okay with that @sfc-gh-xhuang ?

Reply:
Let me see when next release of JDBC is.

Reply:
Even if they release next week, there's no guarantees it will be a simple upgrade either. 
The large strings issue only affects reads and this test, it doesn't affect write behavior which is the only thing we use

Reply:
+1. We are just a bystander for that bug. It is not in our code and they were able to reproduce it themselves. I support not waiting for that fix to land.

Reply:
JDBC plans to release next week but please consider the trade off of trying to use 3.14.6 vs today's 3.14.5